week1/My-own-modified/index.js

```diff
@@ -0,0 +1,43 @@
+"use strict";
+const http = require("http");
+const server = http.createServer((req, res) => {
+  console.log("Creating Server");
+});
+let state = 10;
+let original = 10;
+server.on("request", (req, res) => {
+  console.log(req.method, req.url);
+  switch (req.url) {
+    case "/state":
+      res.setHeader("content-type", "text/html");
+      res.write(`<h1>${state}</h1>`);
+      res.end()
```

Comment:
You call `res.setHeader`, `res.write`, and `res.end` in almost every one of these cases. How would you refactor this to avoid so much duplication?

Reply:
Done.